model/instance/lifecycle/store.go

```diff
@@ -11,6 +11,8 @@ import (
 )
 
 // Store is an object to store and retrieve magic link codes.
+//
+// TODO: Move to [token.Service] with [token.MagicLink] namespace.
```

Comment:
Is it something you want to do in this PR? Or just a reminder for later?

Reply:
This is for later. I try to keep the changes as little as possible and this is not a hard requirement.